docs/wikidata-game.md

```diff
@@ -0,0 +1,30 @@
+# Introduction
+In order to measure the quality fo the pipeline, and to add references to Wikidata we implemented a version of the distributed game.
+
+The main page of the game platform can be found: [here](https://tools.wmflabs.org/wikidata-game/distributed/)
+
+The code for the platform that the games runs on can be found: [here](https://bitbucket.org/magnusmanske/wikiata-game/src/master/public_html/distributed/)
+
+The version of the game deployed in May 2020 by the Reference Island team can be found: [here](https://tools.wmflabs.org/wikidata-game/distributed/)
+
+# Hosting our Game
+The platform requires an api to be provided that serves it "tiles" that can them be shown to users. The api is also sent feedback about user actions.
+The api also provides a small summary for inclusion on the main page of the game.
```

Comment:
I would rephrase that for reading flow (too much redundancy...): 

"The platforms requires the api to... [tiles] and... [feedback]. In addition, the api provides..."

The other alternative is to make this to a list (human brains and eyes love lists):

"The platform requires [...] for the following purposes:
- [tiles]
- [feedback]
- [summary]"

Reply:
Thanks; the list does indeed read way nicer.